azurerm/helpers/authentication/authorization_token.go

```diff
@@ -0,0 +1,53 @@
+package authentication
+
+import (
+	"fmt"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/adal"
+)
+
+func GetAuthorizationToken(c *Config, oauthConfig *adal.OAuthConfig, endpoint string) (*autorest.BearerAuthorizer, error) {
```

Comment:
Nit: Exported functions should be documented

Reply:
done 👍